sdcm/kafka/kafka_cluster.py

```diff
@@ -0,0 +1,184 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+import logging
+from pathlib import Path
+from functools import cached_property
+
+import requests
+
+from sdcm import cluster
+from sdcm.wait import wait_for
+from sdcm.remote import LOCALRUNNER
+from sdcm.utils.git import clone_repo
+from sdcm.utils.common import get_sct_root_path
+from sdcm.utils.remote_logger import CommandClusterLoggerBase
+from sdcm.kafka.kafka_config import SctKafkaConfiguration
+
+# TODO: write/think more about the consumers
+
+logger = logging.getLogger(__name__)
```

Comment:
Use uppercase for the global objects.
I already tried to find `logger` in method scope.

Reply:
fixed